### PR TITLE
Improve addresses & abi tracking

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

#241 > 1 and 2

### Types of changes
Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### CODE REVIEW

File: packages/tasit-contracts/src/TasitContracts.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import estateRegistryABI from "../abi/EstateRegistry.json";
+import gnosisSafeABI from "../abi/GnosisSafe.json";
+import landProxyABI from "../abi/LANDProxy.json";
```
Comment (pcowgill) [inline packages/tasit-contracts/src/TasitContracts.js]:
```suggestion

import landProxyABI from "../abi/LANDProxy.json";
```

File: packages/tasit-contracts/src/TasitContracts.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+import landRegistryABI from "../abi/LANDRegistry.json";
+import manaTokenABI from "../abi/MANAToken.json";
+import marketplaceABI from "../abi/Marketplace.json";
+import myERC20FullABI from "../abi/MyERC20Full.json";
```
Comment (pcowgill) [inline packages/tasit-contracts/src/TasitContracts.js]:
```suggestion

import myERC20FullABI from "../abi/MyERC20Full.json";
```

File: packages/tasit-contracts/src/TasitContracts.js (hunk 1 above)
Comment (pcowgill) [reply packages/tasit-contracts/src/TasitContracts.js]: (Space before the Decentraland contracts)

File: packages/tasit-contracts/src/TasitContracts.js (hunk 2 above)
Comment (pcowgill) [reply packages/tasit-contracts/src/TasitContracts.js]: (Space before the token contracts)

File: packages/tasit-contracts/src/TasitContracts.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+import myERC721FullABI from "../abi/MyERC721Full.json";
+import sampleContractABI from "../abi/SampleContract.json";
```
Comment (pcowgill) [inline packages/tasit-contracts/src/TasitContracts.js]:
```suggestion

import sampleContractABI from "../abi/SampleContract.json";
```

Comment (pcowgill) [reply packages/tasit-contracts/src/TasitContracts.js]: (Space after the token contracts)

File: packages/tasit-contracts/test/EstateRegistry.js [hunk 4]
```diff
@@ -1,24 +1,31 @@
-const abi = require("../3rd-parties/decentraland/land/build/contracts/EstateRegistry.json")
-  .abi;
-const localAddresses = require("../3rd-parties/decentraland/addresses").local;
-const { LANDRegistry: landAddress, EstateRegistry: address } = localAddresses;
+const TasitContracts = require("../dist/TasitContracts").TasitContracts;
```
Comment (pcowgill) [inline packages/tasit-contracts/test/EstateRegistry.js]: Is this line worth splitting out into two lines with destructuring assignment like we do elsewhere, or is this syntax better when we're doing `require`s?

**Comment by pcowgill** [comment]
@marcelomorgado Looks like my merge of another PR introduced some conflicts here

File: packages/tasit-contracts/test/EstateRegistry.js (hunk 4 above)
Comment (marcelomorgado) [reply packages/tasit-contracts/test/EstateRegistry.js]: I think that's fine with `require()`

Comment (pcowgill) [reply packages/tasit-contracts/test/EstateRegistry.js]: Okay thanks, resolving this

**Comment by marcelomorgado** [comment]
> @marcelomorgado Looks like my merge of another PR introduced some conflicts here

Merge resolved

File: packages/tasit-contracts/abi/LANDProxy.json [hunk 5]
```diff
@@ -185,257 +183,4 @@
...
-  "source": "pragma solidity ^0.4.18;\n\nimport \"../Storage.sol\";\nimport \"../upgradable/Proxy.sol\";\n\n\ncontract LANDProxy is Storage, Proxy {\n}\n",
-  "sourcePath": "/home/cmarcelo/git/TasitSDK/packages/tasit-contracts/decentraland/land/contracts/upgradable/LANDProxy.sol",
-  "ast": {
-    "absolutePath": "/home/cmarcelo/git/TasitSDK/packages/tasit-contracts/decentraland/land/contracts/upgradable/LANDProxy.sol",
-    "exportedSymbols": {
-      "LANDProxy": [
```
Comment (pcowgill) [inline packages/tasit-contracts/abi/LANDProxy.json]: Why were so many lines removed from specifically this ABI?

Comment (marcelomorgado) [reply packages/tasit-contracts/abi/LANDProxy.json]: We are storing only the `abi` arrays of each contract, that file was contained the entire contract `json`.

Comment (pcowgill) [reply packages/tasit-contracts/abi/LANDProxy.json]: Got it. Thanks! Resolving

**Pull request merged**